Mathlib/Analysis/NormedSpace/Multilinear/Curry.lean

```diff
@@ -560,38 +560,38 @@ variable (𝕜 G G') {k l : ℕ} {s : Finset (Fin n)}
 to the space of continuous multilinear maps `G [×k]→L[𝕜] G [×l]→L[𝕜] G'` of `k` variables taking
 values in the space of continuous multilinear maps of `l` variables. -/
 def curryFinFinset {k l n : ℕ} {s : Finset (Fin n)} (hk : #s = k) (hl : #sᶜ = l) :
-    (G[×n]→L[𝕜] G') ≃ₗᵢ[𝕜] G[×k]→L[𝕜] G[×l]→L[𝕜] G' :=
+    (G [×n]→L[𝕜] G') ≃ₗᵢ[𝕜] G [×k]→L[𝕜] G [×l]→L[𝕜] G' :=
   (domDomCongrₗᵢ 𝕜 G G' (finSumEquivOfFinset hk hl).symm).trans
     (currySumEquiv 𝕜 (Fin k) (Fin l) G G')
 
 variable {𝕜 G G'}
 
 @[simp]
-theorem curryFinFinset_apply (hk : #s = k) (hl : #sᶜ = l) (f : G[×n]→L[𝕜] G')
+theorem curryFinFinset_apply (hk : #s = k) (hl : #sᶜ = l) (f : G [×n]→L[𝕜] G')
```

Comment:
I'm happy to rubber-stamp this change (and all its similar versions), arguing that we should fix the pretty-printer if we disagree with this - but can you explain why you do this?

Reply:
My motivation for this choice comes from [this line of code](https://github.com/leanprover-community/mathlib4/blob/4326b6f8a8c933fe90ae5f83a18128e94a1a3703/Mathlib/Topology/Algebra/Module/Multilinear/Basic.lean#L28): someone, probably Sébastien', could type what they wanted and chose to write
```
We introduce the notation `M [×n]→L[R] M'` for the space of continuous `n`-multilinear maps from
`M^n` to `M'`.
```

So that is why I made the call that a space between `M` and `[×n]→L[R]` was reasonable!

Reply:
Makes sense; this sounds reasonable.